x-pack/plugins/cases/docs/openapi/components/schemas/case_response_closed_by_properties.yaml

```diff
@@ -0,0 +1,16 @@
+title: Case response properties for closed_by
+type: object
+nullable: true
+properties:
+  email:
+    type: string
+    nullable: true
+  full_name:
+    type: string
+    nullable: true
+  username:
+    type: string
+    example: elastic
+  profile_uid:
```

Comment:
Not sure if we need to specify it here but this field can be `undefined`

Reply:
In f199a07c79f239af273d90d6d65bc374e1c1ed54, I updated the specification to indicate that all of the properties are required *except* profile_uid.  If that's not what you had in mind, just let me know!